app/src/behaviors/behavior_sensor_rotate_key_press.c

```diff
@@ -20,11 +21,47 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 static int behavior_sensor_rotate_key_press_init(const struct device *dev) { return 0; };
 
+static int on_keymap_binding_pressed(struct zmk_behavior_binding *binding,
+                                     struct zmk_behavior_binding_event event) {
+    uint32_t keycode;
+
+    switch (event.position) {
+    case 1:
+        keycode = binding->param1;
+        break;
+    case 2:
+        keycode = binding->param2;
+        break;
+    default:
+        return -ENOTSUP;
+    }
+
+    return ZMK_EVENT_RAISE(zmk_keycode_state_changed_from_encoded(keycode, true, event.timestamp));
+}
+
+static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
+                                      struct zmk_behavior_binding_event event) {
+    uint32_t keycode;
+
+    switch (event.position) {
```

Comment:
As noted below, using the position like this would make it appear to be the same key position as the second and third keys in the keymap.

Maybe a better way to handle this would be to rebase onto #1275, and then instead of giving the sensor behavior itself press/release functions and needing some way to identify whether a press/release corresponds to clockwise or counterclockwise, you would just queue up a press/release for the wrapped behavior?

Reply:
Ha, I didn't know about #1275, should have looked more carefully on outstanding PRs. #1275 is a better design and it also uses the behavior queue to achieve the same idea, I feel like this PR is redundant and we should probably just close this PR and wait on #1275. WDYT?